Source/Libraries/Adapters/PhasorProtocolAdapters/CommonPhasorServices.cs

```diff
@@ -914,6 +915,8 @@ private void m_frameParser_ConnectionAttempt(object sender, EventArgs e)
     private static TemplatedExpressionParser s_pointTagExpressionParser;
     private static Dictionary<string, DataRow> s_signalTypes;
 
+    private static readonly string[] s_commonVoltageLevels = ["44", "69", "115", "138", "161", "169", "230", "345", "500", "765", "1100"];
```

Comment:
I recommend reversing the order here to help avoid issues, like how `"SHELBY_169KV".IndexOf("69")` would indicate that SHELBY is 69 kV instead of 169 kV.

```suggestion
    private static readonly string[] s_commonVoltageLevels = ["1100", "765", "500", "345", "230", "169", "161", "138", "115", "69", "44"];
```